docs/source/llms/langchain/notebooks/langchain-retriever.ipynb

```diff
@@ -76,16 +76,36 @@
     "from bs4 import BeautifulSoup\n",
     "from langchain.chains import RetrievalQA\n",
     "from langchain.document_loaders import TextLoader\n",
-    "from langchain.embeddings.openai import OpenAIEmbeddings\n",
-    "from langchain.llms import OpenAI\n",
     "from langchain.text_splitter import CharacterTextSplitter\n",
     "from langchain.vectorstores import FAISS\n",
+    "from langchain_openai import OpenAI, OpenAIEmbeddings\n",
     "\n",
     "import mlflow\n",
     "\n",
     "assert \"OPENAI_API_KEY\" in os.environ, \"Please set the OPENAI_API_KEY environment variable.\""
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Notebook compatibility\n",
+    "\n",
```

Comment:
nit: similar to above, I think this could be under `Installing Requirements` section